src/insight/metrics/base.py

```diff
@@ -68,7 +68,8 @@ def _add_to_database(self,
                          dataset_name: str,
                          dataset_rows: int = None,
                          dataset_cols: int = None,
-                         category: str = None):
+                         category: str = None,
+                         run_id: str = None):
```

Comment:
Given that `version` is obtained from the an envrionment variable, does it make sense for run_id to also be obtained in the same way? (See lines 83:84 of this file).

Also I've just noticed that line 81: should probably be removed and replaced with run_id if we leave `run_id` as an argument.